main.js

```diff
@@ -1,42 +1,56 @@
-// Note: This only seems to work in Repl.it for some reason but it does "kind of" work.
 
-
-
-
-
-  var towers = [[3, 2, 1], [], []];
-
-  var count = 0;
-
-  var moveTowers = function(start,end) {
-    var startPeg = start - 1; // so user doesn't have to ask for 0 when they mean 1
-    var endPeg = end - 1;     // so user doesn't have to ask for 0 when they mean 1
-    var emptyEndPeg = towers[endPeg].length === 0; // checks if peg is empty. if it's empty, merging is easier.
-    var notEmptyStartPeg = towers[startPeg].length != 0; // you don't want to waste a move moving an empty start peg into a peg that has value.
-    var lastItemInStartArray = towers[startPeg][towers[startPeg].length - 1]; // Last Item In Start Array
-    var lastItemInEndArray = towers[endPeg][towers[endPeg].length-1]; // Last Item in End Array
-    if (lastItemInStartArray < lastItemInEndArray || emptyEndPeg && notEmptyStartPeg) {
-      towers[endPeg].push(lastItemInStartArray);
-      towers[startPeg].pop();
-      count++;
-    } else {
-      console.log("Move is not legal.");
-    }
-
-    console.log(towers);
-    console.log('Move Count: ' + count);
-    checkWinner();
+let towers = [[3, 2, 1], [], []];
+let count = 0;
+
+moveTowers = (start, end) => {
+  const startPeg = start - 1
+  const endPeg = end - 1
+  const emptyEndPeg = towers[endPeg].length === 0 
+  const notEmptyStartPeg = towers[startPeg].length !== 0 
+  const lastItemInStartArray = towers[startPeg][towers[startPeg].length - 1]
+  const lastItemInEndArray = towers[endPeg][towers[endPeg].length-1]
+  if (lastItemInStartArray < lastItemInEndArray || emptyEndPeg && notEmptyStartPeg) {
+    towers[endPeg].push(lastItemInStartArray);
+    towers[startPeg].pop();
+    count++;
+  } else {
+    console.log("Move is not legal.");
+  }
+  printBoard()
+  checkWinner()
+}
+
+printBoard = () => {
+  const pegs = '---'
+  let i = 1
+  const pegsArrayMap = towers.map(x => i++ + ' ' + pegs + ' ' + x);
+  pegsArrayMap.forEach(function(peg) {
+    console.log(peg);
+  }); 
+  i = 1
+  moveCounter()
+}
+
+checkWinner = () => {
+  if ((towers[1].length === 3) || (towers[2].length === 3)) {
+    console.log('Congratulations, you won! It took you ', count, ' total moves.')
+    resetBoard()
   }
+}
 
-    // Does not run properly
-    var checkWinner = function() {
-      if (towers === [[],[3,2,1],[]] || towers === [[],[],[3,2,1]]) {
-        console.log("You Win!!")
-      };
-      };
+// It felt cleaner to make a function that prints the move count rather than console logging it every time
+moveCounter = () => {
+  console.log(`Move Count: `, count)
+}
 
+resetBoard = () => {
+  console.log('resetting board....')
+  towers = [[3, 2, 1,], [], []]
+  count = 0
+}
 
 
+// uncomment for winning combination
```

Comment:
Uncommenting this comment doesn't _seem_ like it would give you a winning combination. Are there lines below expected to be commented out?

Reply:
lol, the lines below were supposed to be commented out. I've gone ahead and commented out the moves so this comment should make much more sense now. Thanks!